crates/ruff_linter/src/fix/edits.rs

```diff
@@ -562,28 +617,77 @@ x = 1 \
         let program = parse_suite(contents).unwrap();
         let stmt = program.first().unwrap();
         assert_eq!(
-            make_redundant_alias(["x"].into_iter(), stmt),
+            make_redundant_alias(["x"].into_iter().map(std::convert::Into::into), stmt),
```

Comment:
nit: using `::from` in cases like this is usually more readable than using `std::convert::Into::into`

```suggestion
            make_redundant_alias(["x"].into_iter().map(Cow::from), stmt),
```